src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
How does the `include` statement should work? 
I'm asking because I've run the `org.scalameter.examples.TestSuite` and surprisingly only second `include` was executed. It's not the issue with this PR, because I've run it also on the `master` branch and got the same result.